client/main.js

```diff
@@ -0,0 +1,22 @@
+
+const getSentenceButton = document.getElementById("get-sentence-btn")
+const sentenceContainer = document.getElementById("sentence-container")
+
+const serverAddress = "localhost"
+const serverPort = 3000
+const getSentenceRoute = "greateSentence"
+
+const getSentenceURI = `http://${serverAddress}:${serverPort}/${getSentenceRoute}`
```

Comment:
Because you are requesting from the server that served you, you can simply request `/path-in-server` (no need to write the origin).
When doing this, you can change domain-name and ports without updating the client code